pages/index.md

```diff
@@ -52,7 +52,7 @@ redirect_from:
           stroke="var(--body-text)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
       </svg>
       <h3 class="welcome-title h3">Explore Functions</h3>
-      <p class="welcome-content">Learn how to create custom logic, flows and integrations for your business</p>
+      <p class="welcome-content">Learn how to create custom logic, flows, and integrations for your business</p>
```

Comment:
@j9t I don't think this comma is needed here and prefer we stick to the current method of no comma in a list of 3+ before and.